src/components/blog/CommunityLinks.astro

```diff
@@ -0,0 +1,17 @@
+---
+---
+<hr>
+
+<p>As we are open source, you can easily check our work on <a href="https://github.com/interledger/">GitHub</a>. If the work mentioned here inspired you, we welcome your contributions. You can join our <a href="https://communityinviter.com/apps/interledger/interledger-working-groups-slack">community slack</a> or participate in the next <a href="https://docs.google.com/document/u/1/d/1T_Q_eRZtL8GluJR9rVADd7fvkB8mPluMk-d96LM4vlg/edit#heading=h.ybdhwl6k5886">community call</a>, which takes place each second Wednesday of the month.</p>
```

Comment:
I remember seeing in either the Slack conversation or blog review discussions that we wanted this at the footer of every blog post, so this component is now part of the blog layout.